### PR TITLE
Update translation_br.xml

### PR DESCRIPTION
**Pull request by FabioZumbi12**
Adding and updating some new lang text, and fixing some wrong/untraslated strings.

### CODE REVIEW

File: translations/translation_br.xml [hunk 1]
```diff
@@ -15,65 +15,65 @@
...
+    <text name="CP_jobParameters_startAt_unload_siloLoader" text="Descarregar carregador de silo"/>
+    <text name="CP_jobParameters_startAt_bales" text="coletar/embalar fardos"/>
+    <text name="CP_jobParameters_subTitle_laneOffset" text="Distância da fileira:"/>
+    <text name="CP_jobParameters_laneOffset_title" text="Distância da fileira:"/>
+    <text name="CP_jobParameters_laneOffset_tooltip" text="Distância da fileira"/>
+    <text name="CP_jobParameters_laneOffset_center" text="centro"/>
+    <text name="CP_jobParameters_laneOffset_left" text="esquerda(1)"/>
+    <text name="CP_jobParameters_laneOffset_right" text="direita(1)"/>
+    <text name="CP_jobParameters_laneOffset_left_2" text="esquerda(2)"/>
+    <text name="CP_jobParameters_laneOffset_right_2" text="direita(2)"/>
```
Comment (Tensuko) [inline translations/translation_br.xml:54]: You seem to have a very old version.
Revert these changes, as the correct one is "CP_fieldWorkJobParamater..."
Just compare it to the other translations e.g. this https://github.com/Courseplay/Courseplay_FS22/blob/ab4e01d97a9c2f78e5b2a27fdb75ecb312f00144/translations/translation_en.xml#L33

File: translations/translation_br.xml [hunk 2]
```diff
@@ -618,50 +616,27 @@ Para excluir uma rota ou pasta, clique no botão excluir e depois na pasta ou ro
...
+J: Copie e cole um percurso em um veículo diferente.
+     Depois que o botão copiar for pressionado, o percurso poderá ser colado em um veículo sem percurso quando o ícone ficar verde.
+K e L: Clicar no texto redefinirá suas alterações para os valores padrão.
+M: Mostra o curso copiado.
+N: Esta área mostra o deslocamento da pista se você tiver um curso de múltiplas ferramentas carregado para o comboio. 
+     Clicar no texto alterna as posições.
+O: Iniciar/parar a gravação de campos personalizados.
+P: Remove o percurso do veículo.
+P: Altera a visibilidade do percurso entre: ligado, desligado e apenas início/parada.
+R: Remove o curso copiado da área de transferência.
```
Comment (Tensuko) [inline translations/translation_br.xml:639]: Same for this one. Check from this line. https://github.com/Courseplay/Courseplay_FS22/blob/ab4e01d97a9c2f78e5b2a27fdb75ecb312f00144/translations/translation_en.xml#L623

**Review by Tensuko — CHANGES_REQUESTED** [review]

**Comment by FabioZumbi12** [comment]
Oh, thats ok

**Comment by FabioZumbi12** [comment]
Fixed strings for new version on https://github.com/Courseplay/Courseplay_FS22/pull/3330/commits/e28830f861eebf77c886b19fd254de514b8c7961

I saw that the English file of the current version available for download is also out of date, and I based it on it, but now it is like the version you provided me.

File: translations/translation_br.xml [hunk 3]
```diff
@@ -617,49 +617,50 @@ Para renomear, basta clicar no botão renomear e depois ns pasta ou rota. Em seg
...
+    <text name="CP_help_page_minihud_balecollect_title" text="Coletar/embrulhar fardos"/>
     <text name="CP_help_page_minihud_balecollect_text" text="
-A: Fardos restantes no campo.
+R: Fardos restantes no campo.
 B: Tipo de fardos a recolher/embalar.
-C: Deslocamento entre a linha central do trator e a linha central do braço da carregadeira. Pode ser necessário ajustar isso para tratores maiores (por exemplo, com pneus mais largos)."/>
-    <text name="CP_help_page_minihud_siloloader_title" text="Carregar silo"/>
+C: Deslocamento entre a linha central do trator e a linha central do braço da carregadeira. Pode ser necessário ajustar isso para tratores maiores (por exemplo, com pneus mais largos).
+"/>
+    <text name="CP_help_page_minihud_siloloader_title" text="Carregador de Silo"/>
```
Comment (Tensuko) [inline translations/translation_br.xml:653]: Keep our format pls.

**Review by Tensuko — CHANGES_REQUESTED** [review]

**Comment by FabioZumbi12** [comment]
Done. Let me know if need more changes!

File: translations/translation_br.xml (hunk 1 above)
Comment (Admilson) [reply translations/translation_br.xml]: thanks for the review

**Review by Admilson — APPROVED** [review]
